back/engines/commercial/analysis/app/models/analysis/tagging.rb

```diff
@@ -4,11 +4,9 @@
 #
 # Table name: analysis_taggings
 #
-#  id         :uuid             not null, primary key
-#  tag_id     :uuid             not null
-#  input_id   :uuid             not null
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
+#  id       :uuid             not null, primary key
+#  tag_id   :uuid             not null
+#  input_id :uuid             not null
```

Comment:
Unrelated auto-comments for models created by other dev (my run of `db:migrate` created the comments)